controllers/provider-alicloud/pkg/controller/worker/machines_test.go

```diff
@@ -283,21 +308,26 @@ var _ = Describe("Machines", func() {
 				}
 
 				scheme = runtime.NewScheme()
-				_ = apisalicloud.AddToScheme(scheme)
-				_ = alicloudv1alpha1.AddToScheme(scheme)
+				_ = api.AddToScheme(scheme)
+				_ = apiv1alpha1.AddToScheme(scheme)
 				decoder = serializer.NewCodecFactory(scheme).UniversalDecoder()
 
 				workerPoolHash1, _ = worker.WorkerPoolHash(w.Spec.Pools[0], cluster)
 				workerPoolHash2, _ = worker.WorkerPoolHash(w.Spec.Pools[1], cluster)
 
-				workerDelegate = NewWorkerDelegate(c, scheme, decoder, machineImages, chartApplier, "", w, cluster)
-			})
+				dummyUse(cluster)
```

Comment:
what is the purpose if this invocation of `dummyUse(cluster)` ?

Reply:
ahh, this is helper used to get rid of error messages for unused variables. It can and was used during development to keep variables with complex expressions intended for later usage.
The tests have very complex initialization settings. And during the reorg of the tests, it always happend, that such a setting was temporarily not used. To be able to compile and run the tests anyway (without removing all this code) I introduced this helper. It has the advantage (compared to putting it into a comment), that this code is still compiled, error checked and handled by the IDE rename operations.